src/page/AlertingPage.tsx

```diff
@@ -39,7 +39,11 @@ const Alerting = () => {
           Learn more about alerting for Synthetic Monitoring.
```

Comment:
The page doesn't display if the user lacks the `grafana-synthetic-monitoring-app.alerts:read` permission:
![image](https://github.com/user-attachments/assets/8b2b0edb-80ba-4467-86d5-fe88f9d613ec)


However, if the user is missing the `alert.rules.external:read` action, I think it would be better to render the Alerting page and show a message indicating this. Without this message, the user might be unaware of the missing permission and could be confused about why they can't access the page.